### DISCUSSION TITLE
new user: unclear how to use the app

### DISCUSSION THREAD

**Discussion by andrewufrank** [zeapo/Android-Password-Store#3065] [Q&A] [answered]
I seem to have the add running and can store and retrieve  some passwords, but I am not clear how to use it best. 
at the moment, i am asked for the passphrase of the key each time I need a password decoded. using passFF on a desktop, I enter the passphrase once and decoding is possible for a a period without entering the passphrase again. my passphrase is too long for easy typing on the small screen. what are the options?  can I have the passphrase active for a time or till the phone is close the next time? 
is this behaviour the same when biometric is working again? i.e. can I decode passwords with my finger?  I will eventually sync the store with git and enter passwords etc. primarily on a desktop, but look for a relatively easy way to decode passwords.
thank you for help!

**Comment by msfjarvis** [comment, marked as answer]
If you're using the [snapshot builds](https://docs.passwordstore.app/docs/users/release-channels) then the app comes with its own passphrase cache that can be enabled from Settings > PGP settings. If you're on version 1.13.x, OpenKeychain has its own cache setting somewhere that will do the job.

**Reply by andrewufrank** [reply]
thank you - i found the cache setting and enabled it. together with the biometric it seems to work.
